products/internetexplorer.md

```diff
@@ -0,0 +1,50 @@
+---
+permalink: /internetexplorer
```

Comment:
Shall we call this `internet-explorer.md` and have `internetexplorer` as an alternate?

```suggestion
permalink: /internet-explorer
```

We have a mix of products with and without hyphens, but hyphens make things more readable.